### DISCUSSION TITLE
use mutate to create a column from a variable

### DISCUSSION THREAD

**Discussion by lei0zhou** [ibis-project/ibis#8562] [Q&A] [answered]
test = 'abc'
mutate(test = lit(1))

will create a column named 'test', but how to create a column named 'abc'?

**Comment by cpcloud** [comment, marked as answer]
`mutate(**{test: lit(1)})`

**Reply by lostmygithubaccount** [reply]
alternative syntax: `mutate(lit(1).name(test))`

**Comment by cpcloud** [comment]
Let's keep discussions open if they have an answer.

**Reply by cpcloud** [reply]
I believe they are more likely to show up in search engine results if they remain open.
